config/packages/framework_extra.yml

```diff
@@ -0,0 +1,3 @@
+sensio_framework_extra:
```

Comment:
Can this be removed?

Reply:
This is needed for the deprecated thing.